contrib/verifybinaries/test.py

```diff
@@ -8,19 +8,24 @@
 
 def main():
     """Tests ordered roughly from faster to slower."""
+    if subprocess.run("gpg --list-keys | grep 0x90C8019E36C2E964 >/dev/null", shell=True).returncode != 0:
+        print("In order to run tests, you should first import the old release signing key:")
+        print("  gpg --recv-key 01EA5486DE18A882D4C2684590C8019E36C2E964")
```

Comment:
Let's add `--keyserver hkps://keyserver.ubuntu.com` to save the user a Google search for `gpg: key 90C8019E36C2E964: new key but contains no user ID - skipped`.